apiserver/common/networkingcommon/shims.go

```diff
@@ -88,7 +88,7 @@ func NewStateShim(st *state.State) (*stateShim, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return &stateShim{stateenvirons.EnvironConfigGetter{st, m}, st, m}, nil
+	return &stateShim{stateenvirons.EnvironConfigGetter{State: st, Model: m}, st, m}, nil
```

Comment:
Why include the field names in the EnvironConfigGetter but not the stateShim?

Oh, I see - because you've added a new field to EnvironConfigGetter? Ok cool.